content/tutorials/managing-config-and-secrets/index.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "Managing Configuration and Secrets"
+title_tag: "Managing Configuration and Secrets"
+layout: single
+
+# A succinct description of the tutorial. It appears on the Tutorials home and collection pages.
+description: Here is a brief description of what this tutorial's all about.
+
+# A similar description used for search results and social-media previews.
+meta_desc: Here is a brief description of what this tutorial's all about.
+
+# An image for the tutorial. It appears on tutorial page and in social-media previews.
+meta_image: config-secrets-meta.png
+
+# The order in which the tutorial appears in most lists. Order is ascending, so higher numbers
+# mean the tutorial will appear further down the list. Positive integers only.
+weight: 999
+
+# A brief summary of the tutorial. It appears at the top of the tutorial page. Markdown is fine.
+summary: |
+    In Pulumi, an environment is a collection of values intended to capture the configuration values needed to work with a particular environment. These can be raw values like server names, environment types, region names and so on. They can also be sensitive values such as database passwords or service tokens.
```

Comment:
Suggest rephrasing not to use the word "environment". The term `environment` here is confusing because that's an ESC concept that we're not actually using in this tutorial. It also conflates against the more common concept of environment variables. Later in the doc we also specify a config value called `myEnvironment` which is yet another meaning of the word (e.g. "production environment").